src/_pytest/fixtures.py

```diff
@@ -1413,6 +1413,11 @@ def parsefactories(self, node_or_obj, nodeid=NOTSET, unittest=False):
         for name in dir(holderobj):
             # The attribute can be an arbitrary descriptor, so the attribute
             # access below can raise. safe_getatt() ignores such exceptions.
+            # additionally properties are ignored by default to avoid triggering warnings
```

Comment:
For reference, the `safe_getattr()` and comment above was added in 3a0a0c2df93ecfcf4fb09a0f7c24bf3a06d1bae5 to fix #2234. There I wrote

> A better solution would be to completely skip access to all custom descriptors, such that the offending code doesn't even trigger. However I think this requires manually going through the instance and all of its MRO for each and every attribute checking if it might be a proper fixture before accessing it. So I took the easy route here.

So this at least seems to be a step in the right direction. But this was a long time ago.

There's a question if this is any backward compatibility concern here. Is it possible in any way to use a property as a fixture? (I'm not familiar with this code yet).

In any case I think it would be to discuss this change in a separate PR so it can be considered properly.

Reply:
as a breaking change its ok to drop consideration for properties

as methods are descriptors, we cant drop everything

i would prefer if we could set up better separation for fixture lokup